tests/sentry/api/endpoints/test_team_members.py

```diff
@@ -46,19 +47,20 @@ def test_team_members_list_does_not_include_invite_requests(self):
         response = self.get_response(self.org.slug, self.team.slug)
         assert response.status_code == 200
         assert len(response.data) == 2
-        assert response.data[0]["id"] == str(self.team_member.id)
-        assert response.data[1]["id"] == str(pending_invite.id)
+        assert response.data[1]["id"] == str(self.team_member.id)
+        assert response.data[0]["id"] == str(pending_invite.id)
```

Comment:
Did the order on these response entries reverse? If so, is this something we need to update our docs on?